doc/modules/preprocessing.rst

```diff
@@ -299,8 +302,39 @@ This can be confirmed on a independent testing set with similar remarks::
   ... # doctest: +ELLIPSIS +SKIP
   array([ 0.01...,  0.25...,  0.46...,  0.60... ,  0.94...])
 
-It is also possible to map the transformed data to a normal distribution by
-setting ``output_distribution='normal'``::
+Mapping to a Gaussian distribution
+----------------------------------
+
+In many modeling scenarios, normality of the features in a dataset is desirable.
+Power transforms are a family of parameteric transformations that aim to
+map data of any distribution to as close to a Gaussian distribution as possible.
+
+:class:`BoxCoxTransformer` provides one such transformation. The Box-Cox transform is given by:
+
+.. math::
+    y_i^{(\lambda)} =
+    \begin{cases}
+    \dfrac{y_i^\lambda - 1}{\lambda} & \text{if } \lambda \neq 0, \\[8pt]
+    \ln{(y_i)} & \text{if } \lambda = 0,
+    \end{cases}
+
+The transformation is parameterized by :math:`\lambda`, which is determined through
+maximum likelihood estimation. Here is an example of using Box-Cox to map samples
+drawn from a lognormal distribution to a normal distribution::
+
+  >>> boxcox = preprocessing.BoxCoxTransformer()
+  >>> X_lognormal = np.random.RandomState(616).lognormal(size=(3, 3))
+  >>> X_lognormal
+  array([[ 1.28331718,  1.18092228,  0.84160269],
+         [ 0.94293279,  1.60960836,  0.3879099 ],
+         [ 1.35235668,  0.21715673,  1.09977091]])
+  >>> boxcox.fit_transform(X_lognormal)
+  array([[ 0.49024349,  0.17881995, -0.1563781 ],
+         [-0.05102892,  0.58863196, -0.57612415],
+         [ 0.69420008, -0.84857822,  0.10051454]])
+
```

Comment:
Maybe it's worth noting that the normal distribution here is not unit variance and zero mean... (Or s it?) Indeed maybe we should have an option to scale and centre the output according to the MLE mean and std...